src/index.ts

```diff
@@ -5,18 +5,21 @@ import * as fs from 'fs';
 require('dotenv').config();
 export const zero = ethers.constants.Zero;
 
+export interface Reward {
+  _address: string;
+  _amount: BigNumber;
```

Comment:
Piggybacking on kyle's comment -- underscored variables are used as a convention in javascript to indicate that the variable is private, but I don't think would apply to this interface as it is being exported.

Reply:
This was done because when calling allocateRewards() from the contract the type definition comes with underscores:
<img width="378" alt="Screenshot 2022-10-20 at 09 01 34" src="https://user-images.githubusercontent.com/105349716/196878666-e5545b20-d4ba-40ac-aa60-ac329d394f24.png">

So when I remove the underscores, even if I cast the distribution array output of the script to Rewards, it will give me the following error:
_Argument of type 'Rewards' is not assignable to parameter of type 'RewardStruct[]'_

Reply:
@solidoracle  -- should the contract type have underscores in the definition? What is the reason those are underscored?

Reply:
If we import the typechain from the contract it will inherit the types specified in the contract which have the underscores. When we call the contract it will require that same naming